search.js

```diff
@@ -0,0 +1,128 @@
+mainWrapperDiv = document.createElement("div");
+mainWrapperDiv.setAttribute("class", "mainWrapperDiv");
+searchDiv = document.createElement("div");
+searchDiv.setAttribute("class", "searchDiv");
+searchDiv.innerHTML = `<input type="text" class="searchBar" placeholder="Enter your search term here..."/>
```

Comment:
Create this as a Form and it should submit with Enter key as well, instead of manually clicking on the button